extensions/functions_comparison.yaml

```diff
@@ -3,66 +3,119 @@
 scalar_functions:
   -
     name: "not_equal"
-    description: Whether two values are not_equal (nulls are considered not not-equal).
+    description: >
+      Whether two values are not_equal
```

Comment:
While we're at it, we should probably normalize to complete sentences with a period at the end for the descriptions (see also the other old comparison functions). All the new functions are already doing that, I think.